schema/controller/job.json

```diff
@@ -13,8 +13,16 @@
   "additionalProperties": false,
   "properties": {
     "id": {
+      "type": "string",
+      "description": "the job's cluster ID"
+    },
+    "uuid": {
```

Comment:
For backwards compatibility, the idea being that old clients will still be able to post `id` being the cluster ID, and newer clients use `uuid`. Not ideal I know, but I think necessary.